sentinelhub/data_utils.py

```diff
@@ -0,0 +1,166 @@
+"""
+Module with statistics to dataframe transformation.
+"""
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from .exceptions import PandasNotInstalled
+from .time_utils import parse_time
+from .type_utils import JsonDict
+
+
+def _extract_hist(hist_cols: Tuple[str, str], hist_data: List[JsonDict]) -> Dict[str, List[float]]:
```

Comment:
why is `hist_cols` a tuple of strings instead of two parameters? It would probably even be best if the function returns two values (bins and counts) and they are transformed into DF form in the outside function (better typechecking)